src/coins/coins.ts

```diff
@@ -2539,6 +2552,34 @@ export const basicCoins: BasicCoin[] = [
     },
   },
 
+  // Nibiru
+  {
+    key: CoinKey.NIBI,
+    name: 'Nibiru',
+    logoURI:
+      'https://imgproxy-mainnet.routescan.io/WWpIrHMBrbtXh7cze-P7tXuoP5Yq2sQ0NSmuhhHzkNk/pr:thumb_32/aHR0cHM6Ly9jbXMtY2RuLmF2YXNjYW4uY29tL2NtczIvNjkwMF90b2tlbi40ZGNkYTQ1YmU0Yzkuc3Zn',
```

Comment:
this image is too large to have potential display issue. Try to use wnibi's or our repo image

Reply:
This is an SVG :)